xcp/bootloader.py

```diff
@@ -30,7 +30,7 @@
 import copy
 from typing import cast
 
-import branding  # pytype: disable=import-error
+import xcp.branding  # pytype: disable=import-error
```

Comment:
doesn't this fix the import error? looks like the comment can be dropped

Reply:
You're right! I also need to update the references to it, I've pushed a change to fix both of those issues :)